src/Api/HL/Controller/ITILController.php

```diff
@@ -221,6 +221,14 @@ public static function getRawKnownSchemas(): array
 
         $base_task_schema = [
             'type' => Doc\Schema::TYPE_OBJECT,
+            'x-rights-conditions' => [ // Object-level extra permissions
+                'read' => static function () {
+                    if (!\Session::haveRight(\CommonITILTask::$rightname, \CommonITILTask::SEEPRIVATE)) {
+                        return ['WHERE' => ['is_private' => 0]];
```

Comment:
I think that the user should be allowed to see the private tasks he created itself.